slave-dotnet20/Dockerfile.rhel7

```diff
@@ -0,0 +1,37 @@
+FROM registry.access.redhat.com/openshift3/jenkins-slave-base-rhel7
+
+# Labels consumed by Red Hat build service
+LABEL com.redhat.component="jenkins-slave-dotnet20-rhel7-docker" \
+      name="openshift3/jenkins-slave-dotnet20-rhel7" \
+      version="3.6" \
+      architecture="x86_64" \
+      release="1" \
+      io.k8s.display-name="Jenkins Slave .NET Core 2.0" \
+      io.k8s.description="The jenkins slave dotnet image has the dotnet tools on top of the jenkins slave base image." \
+      io.openshift.tags="openshift,jenkins,slave,dotnet,dotnet20"
+
+# Don't download/extract docs for nuget packages
+# Don't do initially populate of package cache
+# Enable nodejs and dotnet scl
+ENV DOTNET_CORE_VERSION=2.0 \
+    BASH_ENV=/usr/local/bin/scl_enable \
+    ENV=/usr/local/bin/scl_enable \
+    PROMPT_COMMAND=". /usr/local/bin/scl_enable" \
+    ENABLED_COLLECTIONS="rh-nodejs6 rh-dotnet20" \
+    NUGET_XMLDOC_MODE=skip \
+    DOTNET_SKIP_FIRST_TIME_EXPERIENCE=1
+
+COPY contrib/bin/scl_enable /usr/local/bin/scl_enable
+
+# Install
+RUN INSTALL_PKGS="$ENABLED_COLLECTIONS" && \
```

Comment:
Would we want to install nss_wrapper? Will there be a problem with resolving usernames with random IDs? Should we install the full `rh-nodejs6` collection or just `rh-nodejs6-npm`? We use the latter for our main builder image.

Reply:
We want the nodejs-npm package as the s2i builder. nss_wrapper question came up on the openshift/jenkins PR and is not needed in the jenkins slave images.